cacti/README.md

```diff
@@ -9,109 +9,157 @@ Get metrics from cacti service in real time to:
-init_config:
-
-instances:
-    -   mysql_host: localhost
-        mysql_user: datadog
-        mysql_password: hx3beOpMFcvxn9gXcs0MU3jX
-        rrd_path: /path/to/cacti/rra
-        #field_names:
-        #    - ifName
-        #    - dskDevice
-        #    - ifIndex
-        #rrd_whitelist: /path/to/rrd_whitelist.txt
-```
+    mysql -u datadog --password=<MYSQL_PASSWORD> -D cacti -e "select * from data_template_data limit 1" && \
+    echo -e "\033[0;32mMySQL grant - OK\033[0m" || \
+    echo -e "\033[0;31mMissing SELECT grant\033[0m"
+    ```
 
-Give the datadog-agent user access to the RRD files
+3. Give the `datadog-agent` user access to the RRD files:
 
-```shell
-sudo gpasswd -a dd-agent www-data
-sudo chmod -R g+rx /var/lib/cacti/rra/
-sudo su - datadog-agent -c 'if [ -r /var/lib/cacti/rra/ ];
-then echo -e "\033[0;31mdatadog-agent can read the RRD files\033[0m";
-else echo -e "\033[0;31mdatadog-agent can not read the RRD files\033[0m";
-fi'
-```
+    ```shell
+    sudo gpasswd -a dd-agent www-data
+    sudo chmod -R g+rx /var/lib/cacti/rra/
+    sudo su - datadog-agent -c 'if [ -r /var/lib/cacti/rra/ ];
+    then echo -e "\033[0;31mdatadog-agent can read the RRD files\033[0m";
+    else echo -e "\033[0;31mdatadog-agent can not read the RRD files\033[0m";
+    fi'
+    ```
+
+#### Host
+
+Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized](#containerized) section.
+
+1. Configure the Agent to connect to MySQL, edit your `cacti.d/conf.yaml` file. See the [sample cacti.d/conf.yaml][2] for all available configuration options:
+
+    ```yaml
+        init_config:
+
+        instances:
+
+            ## @param mysql_host - string - required
+            ## url of your MySQL database
+            #
+          - mysql_host: "localhost"
+
+            ## @param mysql_port - integer - optional - default: 3306
+            ## port of your MySQL database
+            #
+            # mysql_port: 3306
+
+            ## @param mysql_user - string - required
+            ## User to use to connect to MySQL in order to gather metrics
+            #
+            mysql_user: "datadog"
+
+            ## @param mysql_password - string - required
+            ## Password to use to connect to MySQL in order to gather metrics
+            #
+            mysql_password: "<MYSQL_PASSWORD>"
+
+            ## @param rrd_path - string - required
+            ## The Cacti checks requires access to the Cacti DB in MySQL and to the RRD
+            ## files that contain the metrics tracked in Cacti.
+            ## In almost all cases, you'll only need one instance pointing to the Cacti
+            ## database.
+            ## The `rrd_path` will probably be `/var/lib/cacti/rra` on Ubuntu
+            ## or `/var/www/html/cacti/rra` on any other machines.
+            #
+            rrd_path: "<CACTI_RRA_PATH>"
+    ```
+
+2. [Restart the Agent][3].
+
+#### Containerized
```

Comment:
Where do we say that they should build their own agent docker image to support this integration?